src/app/models/__tests__/workspace.server.model.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { ObjectId } from 'bson-ext'
+import { omit } from 'lodash'
+import mongoose from 'mongoose'
+
+import { getWorkspaceModel } from 'src/app/models/workspace.server.model'
+import { IUserSchema } from 'src/types'
+
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+
+const Workspace = getWorkspaceModel(mongoose)
+
+const MOCK_USER_ID = new ObjectId()
+const MOCK_FORM_ID = new ObjectId()
+const MOCK_WORKSPACE_ID = new ObjectId()
+const MOCK_WORKSPACE_FIELDS = {
+  _id: MOCK_WORKSPACE_ID,
+  title: 'Workspace1',
+  admin: MOCK_USER_ID,
+  formIds: [],
+}
+
+describe('Workspace Model', () => {
+  let FORM_COLLAB_READ_ONLY_USER: IUserSchema
+  let FORM_COLLAB_READ_AND_WRITE_USER: IUserSchema
+  let FORM_ADMIN_USER: IUserSchema
+
+  beforeAll(async () => await dbHandler.connect())
+  beforeEach(async () => {
+    const { form, user: adminUser } = await dbHandler.insertEncryptForm({
+      formId: MOCK_FORM_ID,
+      userId: MOCK_USER_ID,
+    })
+    const { user: collabReadOnlyUser } =
+      await dbHandler.insertFormCollectionReqs({
+        userId: new ObjectId(),
+        mailName: 'readOnly',
+      })
+    const { user: collabReadAndWriteUser } =
+      await dbHandler.insertFormCollectionReqs({
+        userId: new ObjectId(),
+        mailName: 'readAndWrite',
+      })
+    await form.updateFormCollaborators([
+      {
+        email: collabReadOnlyUser.email,
+        write: false,
+      },
+      {
+        email: collabReadAndWriteUser.email,
+        write: true,
+      },
+    ])
+
+    await Workspace.create(MOCK_WORKSPACE_FIELDS)
+    FORM_ADMIN_USER = adminUser
+    FORM_COLLAB_READ_AND_WRITE_USER = collabReadAndWriteUser
+    FORM_COLLAB_READ_ONLY_USER = collabReadOnlyUser
+  })
+  afterEach(async () => await dbHandler.clearDatabase())
+  afterAll(async () => await dbHandler.closeDatabase())
+
+  describe('Schema', () => {
+    it('should create and save successfully', async () => {
+      const { user: existentUser } = await dbHandler.insertFormCollectionReqs({
+        userId: new ObjectId(),
+        mailName: 'userThatExists',
+      })
+      const expectedWorkspaceObject = {
+        title: 'Workspace2',
+        admin: existentUser._id,
+        formIds: [],
+      }
+      const validWorkspace = new Workspace(expectedWorkspaceObject)
+      const saved = await validWorkspace.save()
+      const actualSavedObject = omit(saved.toObject(), [
+        '_id',
+        'created',
+        'lastModified',
+        '__v',
+      ])
+
+      expect(saved._id).toBeDefined()
+      expect(saved.created).toBeInstanceOf(Date)
+      expect(saved.lastModified).toBeInstanceOf(Date)
+      expect(actualSavedObject).toEqual(expectedWorkspaceObject)
+    })
+
+    it("should update workspace's formIds successfully because user is the form admin", async () => {
```

Comment:
this doesn't describe the test case clearly - the action is not an update, and "because user is the form admin" is unclear. maybe "should save successfully when workspace admin is the same as form admin"?